worker/src/main.rs

```diff
@@ -69,14 +70,14 @@ struct Diffoscope {
     pub b: PathBuf,
 }
 
-async fn spawn_rebuilder_script_with_heartbeat<'a>(client: &Client, distro: &Distro, item: &QueueItem, config: &config::ConfigFile) -> Result<Rebuild> {
-    let input = item.package.url.to_string();
+async fn spawn_rebuilder_script_with_heartbeat<'a>(client: &Client, distro: &Distro, privkey: &'a PrivateKey, item: &QueueItem, config: &config::ConfigFile) -> Result<Rebuild> {    let input = item.package.url.to_string();
```

Comment:
Should `let input...` be in the next line? I sense some editor shenanigans. :smile:

Reply:
Hahaha good catch! Indeed, I will fix the formatting for this.